integration/simulation/params/params.go

```diff
@@ -39,6 +39,8 @@ type SimParams struct {
 
 	StartPort int  // The port from which to start allocating ports. Must be unique across all simulations.
 	IsInMem   bool // Denotes that the sim does not have a full RPC layer.
+
+	Timeout int // How long to wait for transactions to be confirmed.
```

Comment:
Don't want to be pernickety but could we call it `ReceiptTimeout` or `TxTimeout` because it's a bit vague at simulation config level. Also probably should put `Millis` on the name or use a `Duration`.

Reply:
Fixed